extensions/amp-image-slider/0.1/test/integration/test-amp-image-slider.js

```diff
@@ -116,7 +116,7 @@ t.run('amp-image-slider', function () {
         expect(win.getComputedStyle(s1.leftLabel)['padding']).to.equal('16px');
       });
 
-      describe('using mouse', () => {
+      describe.skip('using mouse', () => {
```

Comment:
Can you file an issue and add a TODO here?

Reply:
In progress

Reply:
(was looking for the relevant template, realized it's just a label)

Reply:
(looking up who might be related to the test because most lines have you as the blame from a Prettier PR)

Reply:
+1 to adding a TODO with an issue / owner to unskip this in the future

Reply:
Thanks for doing this.